worker/modelcache/manifold.go

```diff
@@ -5,7 +5,7 @@ package modelcache
 
 import (
 	"github.com/juju/errors"
-	"github.com/juju/pubsub"
+	"github.com/juju/pubsub/v2"
```

Comment:
Does dependency engine context.Get() support passing into an interface? This is the sort of thing that feels like instead of:
```
var hub *pubsub.StructuredHub
if err := context.Get(..., &hub)
```
Into a different way of grabbing the instance of an interface that we need.

That isn't really *this* patch as much as it is a factor for something we should be considering for patterns that we use.